src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1622,7 +1613,7 @@ UsgsAstroLsSensorModel::getValidImageRange() const
 {
    return std::pair<csm::ImageCoord, csm::ImageCoord>(
       csm::ImageCoord(0.0, 0.0),
-      csm::ImageCoord(m_nLines, m_nSamples));
+      csm::ImageCoord(m_nLines, m_nSamples)); // Technically nl and ns are outside the image in a zero based system.

```

Comment:
per the CSM API docs, this should be the range over which the sensor model is valid and not just the physical image. The API docs say we should return =- 99,999 since there is no exact limit.

The true limit on this would be based on the distortion model, which is hard to generalize.

Reply:
@jessemapel Should this function be changed to reflect that the valid range is more dynamic? I guess I'm not sure exactly what should be done about this function in this PR specifically

Reply:
For right now, we should return the -99,999 default. We could eventually do some computations to find the domain over which the distortion functions are invertible, but that would be way out of scope on this.

Reply:
This change is outside the scope of this PR though, see #236